app/src/main/java/io/intrepid/contest/rest/ContestWrapper.java

```diff
@@ -0,0 +1,15 @@
+package io.intrepid.contest.rest;
+
+import com.google.gson.annotations.SerializedName;
+
+import io.intrepid.contest.models.Contest;
+
+public class ContestWrapper {
+
+    @SerializedName("contest")
```

Comment:
No need for this when the name matches the Gson factory from the `RetrofitClient`